src/signframework/keygen.c

```diff
@@ -59,7 +59,7 @@ int main(int argc, char** argv)
     const char	*pubKeyFilename;	/* destination file for the generated public key */
     const char	*userName;		/* optional user name */
     const char 	*password;		/* optional user password */
-    unsigned int bitSize = 2048;	/* default RSA key size */
+    unsigned int bitSize = 4096;	/* default RSA key size */
```

Comment:
because github sucks I can't make the comment where it makes sense but the printUsage function at the bottom mentions 2048 twice so needs to be updated as well